java/src/main/java/gov/doc/ntia/sigmf/MetaDoc.java

```diff
@@ -101,19 +93,19 @@ public void addCapture(Capture capture) {
   }
 
   /**
-   * Creates an Acquisition (MetaDoc + Data) by reading the data file in
-   * Global dataFilePath.
+   * Creates an Acquisition (MetaDoc + Data) by reading the data file in Global dataFilePath.
```

Comment:
dataFilePath may need to be changed depending on what changes get made in Global.java

Reply:
Updated to use core:dataset